src/Components/Nav/Nav.js

```diff
@@ -0,0 +1,117 @@
+import { Link, useLocation } from 'react-router-dom';
+import { withRouter } from 'react-router-dom/cjs/react-router-dom.min';
+import styled from 'styled-components';
+
+function Nav() {
+  let locatioin = useLocation();
```

Comment:
```suggestion
  const location = useLocation();
```
- 변수는 가급적 const로 선언해주세요
- 오타 있습니다! location으로 바꿔주세요

Reply:
수정했습니다.